specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -161,18 +161,25 @@ void RegisterForActivation()
 {
     // Register one or more supported image filetypes, an icon,
     // and zero or more verbs for the File Explorer context menu.
+    string imageFileTypes[3] = { ".jpg", ".png", ".bmp" };
+    string verbs[2] = { "view", "edit" };
     ActivationRegistrationManager::RegisterForFileTypeActivation(
-        new string[]{ ".jpg", ".png", ".bmp" },
-        "MyResources.dll",
-        12,
-        new string[]{ "view", "edit" });
+        imageFileTypes,
+        "MyResources.dll", 0,
+        verbs);
 
-    // Register another set of filetypes, with a different
-    // icon, and no specified verbs.
+    // Register another set of filetypes, falling back to the
+    // default icon, and no specified verbs.
+    string videoFileTypes[3] = { ".mov", ".wmv", ".mp3" };
     ActivationRegistrationManager::RegisterForFileTypeActivation(
```

Comment:
Where are the Startup Task register calls?

Reply:
Further down in the doc.